bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/manager/ShellyManagerPage.java

```diff
@@ -345,7 +345,7 @@ private void addAttribute(Map<String, String> properties, ShellyManagerInterface
                         value = dateTimeState.format(null).replace('T', ' ').replace('-', '/');
                         break;
                     default:
-                        value = dateTimeState.format(null).replace('T', ' ').replace('-', '/');
+                        value = getTimestamp(dateTimeState);
```

Comment:
Why this change here, but not in line 345? What is the desired format, do you actually want to use `DateTimeType` for formatting (openHAB), or `ZonedDateTime` (Java)?

In any case, not related to adding new device support, and looks like a half overwrite of 03a0993b995e8f295e1521b063ae6be16c2e24cb.